docs/docs/APIdocs/layers/nn/Non-linear Activations/SoftMin.md

```diff
@@ -0,0 +1,79 @@
+## SoftMin ##
+
+**Scala:**
+```scala
+val sm = SoftMin()
+```
+**Python:**
+```python
+sm = SoftMin()
+```
+
+Applies the SoftMin function to an n-dimensional input Tensor, rescaling them so that the
```

Comment:
Them means every dimension. For 2-dim input tensor, it will output 2-dim tensor too. For every dimension (row), elements values lie in range(0, 1), and the sum is 1, like the example.